Makefile

```
@@ -132,6 +132,10 @@ evg-test-atlas-data-lake:
 evg-test-enterprise-auth:
 	go run -tags gssapi ./cmd/testentauth/main.go
 
+.PHONY: evg-test-oidc-auth
+evg-test-oidc-auth:
+	go run ./cmd/testoidcauth/main.go
```

Since we're running this not as part of a standard test suite, we need to explicitly add a 2nd run with the data race detector enabled.

```suggestion
	go run ./cmd/testoidcauth/main.go
	go run -race ./cmd/testoidcauth/main.go
```

P.S. Yes, we should also do this for all of the other `go run ...` make targets, but we can do that in another PR.